graph/tags.go

```diff
@@ -231,10 +240,12 @@ func (store *TagStore) Delete(repoName, ref string) (bool, error) {
 	return deleted, store.save()
 }
 
+// Tag a image.
```

Comment:
"Tag adds the specified repoName:tag to the given image." or something like that.